pkg/utils/tekton/controller.go

```diff
@@ -539,3 +541,33 @@ func (k KubeController) GetRekorHost() (rekorHost string, err error) {
 	}
 	return
 }
+
+func (s *SuiteController) CreatePVCAccessMode(pvcName string, namespace string, accessMode corev1.PersistentVolumeAccessMode) (*corev1.PersistentVolumeClaim, error) {
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pvcName,
+			Namespace: namespace,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				accessMode,
+			},
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse("1Gi"),
+				},
+			},
+		},
+	}
+
+	createdPVC, err := s.K8sClient.KubeInterface().CoreV1().PersistentVolumeClaims(namespace).Create(context.TODO(), pvc, metav1.CreateOptions{})
+	if err != nil {
+		klog.Error((err))
+		return nil, err
+	}
+	return createdPVC, err
+}
+
+func (s *SuiteController) GetPipelineRunInNamespace(namespace string) (*v1beta1.PipelineRunList, error) {
```

Comment:
can we have a comment for this? Also this works without a pipeline name, just namespace?

Reply:
I also don't see this used in `tests/release/release.go` what do we need it for?